CMakeLists.txt

```diff
@@ -20,77 +20,9 @@ include(cmake/target_default_compile_options.cmake)
 # collect all include and source files
 # (use the manual list to improve ide support)
```

Comment:
No longer true :)

Reply:
That's true!

Reply:
Oh, ok, sorry, I was thinking you meant You are using the manual list, but you mean it for the user to replace the list with the manual one, yes it is true